### ISSUE TITLE
Files not in cwd

### ISSUE THREAD

**Issue by knausb** [grunwaldlab/Population_Genetics_in_R#36]
New users may attempt to open files when r is not in the correct working directory. We should explain what the current working directory is and how to query the files in it.

**Comment by zkamvar** [comment]
A good way to do this is to use the `here` package. That way they can use a file anchor point such as an rstudio project, git repo, or an empty file called “.here”

**Comment by knausb** [comment]
I'm not familiar with `here`, I'll look into it. My usual is to suggest `getwd()` and `list.files()` to determine where they are.

**Comment by grunwald** [comment]
Alternative would be to download the files from within R if workshop has WiFi. That should get around it?

**Comment by zkamvar** [comment]
@knausb, I found this blogpost to be a good introduction behind the rational for using `here`: https://www.tidyverse.org/articles/2017/12/workflow-vs-script/

**Comment by knausb** [comment]
Thanks @zkamvar for a nice post! I can't count the number of times my scripts began with

```
setwd("C:\Users\jenny\path\that\only\I\have")
rm(list = ls())
```

in my pre-Rstudio days. I think the concept of an Rstudio Project circumvents this nicely. I'm not sure I appreciate the beauty of `here` yet. At the present, it seems like a reinvention of existing functions to me. I'll try to work on a PR to propose a solution, sometimes doing illustrates things.

**Comment by zkamvar** [comment]
I think the main benefit of here is you can have a more organized folder structure for your scripts and figures since here will search up the directory tree until it finds the root of the project.

So, if you have a script in scripts/plotThis.Rmd, and you want to write a plot to figs/thisPlot.svg, you can use:

svg(here::here("figs", "thisPlot.svg")
plot(x)
dev.off()

Inside of the script and it will work wherever you are in the directory tree.

**Comment by knausb** [comment]
Addressed at: ff36c92eb8532886e24740d9d976546736345a69
